metadata-jobs/mae-consumer/src/main/java/com/linkedin/metadata/kafka/hook/GraphIndexUtils.java

```diff
@@ -56,11 +56,9 @@ private static List<Map<String, Object>> getPropertiesList(@Nullable final Strin
   @Nullable
   private static boolean isValueListValid(@Nullable final List<?> entryList, final int valueListSize) {
     if (entryList == null) {
-      log.warn("Unable to get entry as entryList is null");
```

Comment:
removed as these don't really help anywhere and spam the logs in case of ingestions